Adds Dissolvable Cleaner Tablets

HanSolo1519 opened the pull request:
Adds Idris-themed space cleaner pills (to dissolve in water (or eat, I'm not your boss)) and boxes for them.

SleepyGemmy commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -1230,3 +1230,12 @@
 	. = ..()
 	make_exact_fit()
 
+/obj/item/storage/box/spacecleaner
+	name = "Idris advamced dissolvable cleaner tablets"
+	desc = "A box of advanced chemical tablets designed by Idris Incorporated. A new generation of cleaning chemicals. Instructions on the box read; dissolve tablet fully in bucket of water, do not consume tablet, do not consume cleaner water from concentration."
+	icon = 'icons/obj/tabletcase.dmi'
+	icon_state = "tabletcase"
+	max_storage_space = 16
+	starts_with = list(
+		/obj/item/reagent_containers/pill/spacecleaner = 16,
+	)
```
```suggestion
/obj/item/storage/box/cleaner_tablets
	name = "\improper Idris Incorporated cleaner tablets case"
	desc = "A box of advanced formula chemical tablets designed by Idris Incorporated."
	desc_extended = "A new generation of cleaning chemicals, according to Idris at least. The instructions on the box reads: \"Dissolve tablet fully in container of water\". A warning label mentions that you should not consume the tablets nor drink the mixture after dissolving them."
	icon = 'icons/obj/tabletcase.dmi'
	icon_state = "tabletcase"
	max_storage_space = 16
	starts_with = list(
		/obj/item/reagent_containers/pill/cleaner_tablet = 16
	)
```

SleepyGemmy commented on code/modules/reagents/reagent_containers/pill.dm:
```diff
@@ -333,3 +333,11 @@
 	icon_state = "pill8"
 	reagents_to_add = list(/singleton/reagent/steramycin = 5)
+
+/obj/item/reagent_containers/pill/spacecleaner
+	name = "dissolvable cleaner tablet"
+	desc = "A highly advanced dissolvable tablet laden with the cutting edge of Idris chemical science. Smells like dried bleach. It's probably just bleach."
+	icon = 'icons/obj/cleanertablet.dmi'
+	icon_state = "cleanertablet"
+	reagents_to_add = list(singleton/reagent/spacecleaner = 15)
```
```suggestion
/obj/item/reagent_containers/pill/cleaner_tablet
	name = "dissolvable cleaner tablet"
	desc = "An advanced formula dissolvable tablet laden with the cutting edge of Idris chemical science. Smells like dried bleach. It's probably just bleach."
	icon = 'icons/obj/cleanertablet.dmi'
	icon_state = "cleanertablet"
	reagents_to_add = list(singleton/reagent/spacecleaner = 15)
```

HanSolo1519 commented:
Thanks gem you're a major help.

I still have to compile, test, and map the boxes into the janitorial closets (and probably junk loottable, now that I'm thinking about it). I'm having computer issues though so it may be a couple days.

alsoandanswer reviewed (changes requested):
Don't make an entirely new .dmis for singular sprites, use the regular box and illustration system for the box and put the pill in chemical.dmi

SleepyGemmy commented:
> I still have to compile, test, and map the boxes into the janitorial closets [...]

make sure you add it to the lockers in the code and not map them in.

HanSolo1519 commented:
> Don't make an entirely new .dmis for singular sprites, use the regular box and illustration system for the box and put the pill in chemical.dmi

I was explicitly told not to, for some reason. I'm not objected to changing, but I want to make sure that I'm actually supposed to after I asked and was told otherwise earlier.
![image](https://github.com/Aurorastation/Aurora.3/assets/72716882/4e6beb2c-b27c-4d62-830b-f30215d5caa2)
![image](https://github.com/Aurorastation/Aurora.3/assets/72716882/6ca83e34-07d4-4b35-b721-ecec55de2c5c)
![image](https://github.com/Aurorastation/Aurora.3/assets/72716882/db49833a-9857-455f-998c-2bb6bed7887a)

alsoandanswer commented:
> > Don't make an entirely new .dmis for singular sprites, use the regular box and illustration system for the box and put the pill in chemical.dmi
> 
> I was explicitly told not to, for some reason. I'm not objected to changing, but I want to make sure that I'm actually supposed to after I asked and was told otherwise earlier. ![image](https://user-images.githubusercontent.com/72716882/276753042-4e6beb2c-b27c-4d62-830b-f30215d5caa2.png) ![image](https://user-images.githubusercontent.com/72716882/276753104-6ca83e34-07d4-4b35-b721-ecec55de2c5c.png) ![image](https://user-images.githubusercontent.com/72716882/276753190-db49833a-9857-455f-998c-2bb6bed7887a.png)

Well, for clothes, they use contained sprites, same with custom items. So they have their own .dmis.

However these should be fine

HanSolo1519 commented:
Moved the pill icon to chemical.dm
Replaced Lizzy's sprite (Thank you for the help though!) with a normal box illustration in line with the rest of them.
Added a box of cleaner pills to the custodian locker startlist.
Added the pill and pillbox to the maintenance loottable (I think)

Still need to get build.bat to not error with windows when I go to test it, but other than that it should be good. I'm not going to finalize this PR until I can make sure it works though.

NonQueueingMatt commented:
Reopen when conflicts are resolved. And make sure you tag your request as a finished one, not as a draft
